src/Analyser/NodeScopeResolver.php

```diff
@@ -921,8 +921,12 @@ private function processStmtNode(
 			}
 		} elseif ($stmt instanceof Static_) {
 			$hasYield = false;
+			$comment = CommentHelper::getDocComment($stmt);
 			foreach ($stmt->vars as $var) {
 				$scope = $this->processStmtNode($var, $scope, $nodeCallback)->getScope();
+				if ($comment !== null) {
+					$scope = $this->processVarAnnotation($scope, $var->var->name, $comment, false);
```

Comment:
This fails in PHP_CodeSniffer. Run the whole build locally with `vendor/bin/phing` :)

Reply:
I honestly hate the `EarlyExitNotUsed` rule 😅 Sometimes the flow makes more sense this way.

> Run the whole build locally

Yeah sorry I forgot 😆 I get too eager creating merge requests.